src.ts/providers/json-rpc-provider.ts

```diff
@@ -60,6 +60,10 @@ export class JsonRpcProvider extends Provider {
         return this.sendJsonRpc('chunk', [chunkId]);
     }
 
+    async gasPrice(blockId: BlockId|null): Promise<GasPrice> {
+        return this.sendJsonRpc('Gas Price', [blockId])
```

Comment:
well, if you add a test you will see that this is not the way to call this rpc. I believe that it should be 'gas_price', but please do add a test to make sure that this works properly.